src/overlays/actors/ovl_En_Syateki_Crow/z_en_syateki_crow.c

```diff
@@ -243,30 +238,34 @@ void func_809CABC0(EnSyatekiCrow* this, PlayState* play) {
         this->actor.shape.rot.z += 0x1780;
     }
 
-    if (this->unk_1C4 > 20) {
-        func_800B3030(play, &this->actor.world.pos, &D_809CB050, &D_809CB050, this->actor.scale.x * 10000.0f, 0, 0);
+    if (this->deathTimer > 20) {
+        func_800B3030(play, &this->actor.world.pos, &sZeroVec, &sZeroVec, this->actor.scale.x * 10000.0f, 0, 0);
         syatekiMan->guayHitCounter++;
         syatekiMan->guayFlags &= ~(1 << EN_SYATEKI_CROW_GET_NUMBER(&this->actor));
-        func_809CA5D4(this);
+        EnSyatekiCrow_SetupWaitForSpawn(this);
     }
 
-    this->unk_1C4++;
+    this->deathTimer++;
 }
 
-void func_809CACD0(EnSyatekiCrow* this, PlayState* play) {
-    if (this->actionFunc == func_809CA8E4) {
-        if (this->unk_23C.base.acFlags & AC_HIT) {
+static Vec3f sSsExtraVelocity = { 0.0f, 20.0f, 0.0f };
+
+static Vec3f sSsExtraAcceleration = { 0.0f, 0.0f, 0.0f };
+
+void EnSyatekiCrow_UpdateDamage(EnSyatekiCrow* this, PlayState* play) {
+    if (this->actionFunc == EnSyatekiCrow_Fly) {
+        if (this->collider.base.acFlags & AC_HIT) {
             play_sound(NA_SE_SY_TRE_BOX_APPEAR);
-            this->unk_1C4 = 0;
-            this->unk_23C.base.acFlags &= ~AC_HIT;
-            EffectSsExtra_Spawn(play, &this->actor.world.pos, &D_809CB0C0, &D_809CB0CC, 5, 1);
-            func_809CAAF8(this);
+            this->deathTimer = 0;
+            this->collider.base.acFlags &= ~AC_HIT;
+            EffectSsExtra_Spawn(play, &this->actor.world.pos, &sSsExtraVelocity, &sSsExtraAcceleration, 5, 1);
```

Comment:
What's the motivation for having `Extra` in the velocity/acceleration names? Also, I think `Accel` should be enough/is more consistent?

Reply:
The symbol name is `s` (for static) + `SsExtra` (named after the effect) + `Velocity/Accel`. I suppose I can just drop the effect name from it, though.